en/docs/learn/configuring-iwa-on-linux.md

```diff
@@ -61,11 +61,11 @@ or federated authentication in WSO2 Identity Server (IS).
 		Kerberos does not work with IP addresses, it relies on
 		domain names and correct DNS entries only.
 
-2.  Open the <IS_HOME>/repository/conf/deployment.toml file and add the following configuration.
+2.  Open the `<IS_HOME>/repository/conf/deployment.toml` file and add the following configuration.
 
     ``` xml
-   		[server]
-		hostname="idp.wso2.com"
+    [server]
+	hostname="idp.wso2.com"
```

Comment:
shall we remove the extra spaces before hostname and bring it to the same level as [server]?

Reply:
Updated the PR with the changes.